sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -908,12 +909,16 @@ def test_session_by_session_client_conn_str_receive_handler_peeklock_abandon(sel
                     message = Message("Handler message no. {}".format(i), session_id=session_id)
                     sender.send(message)
 
-            with sb_client.get_queue_session_receiver(servicebus_queue.name, session_id=session_id, prefetch=0) as receiver:
+            with sb_client.get_queue_session_receiver(servicebus_queue.name, session_id=session_id, prefetch=0, idle_timeout=1) as receiver:
```

Comment:
is idle_timeout being 1s too short?

Reply:
It's a reasonable question.  Tangentially, it reminds me that I'd give my an arm and a leg for a test set that wasn't 100% integration tests (of which I'm largely culpable for); since timeout tuning is a really bad "test-code-smell."  To your point, however, I didn't run into this locally, and would HOPE it would be sufficient since we send multiple messages and thus would have already given some time for them to "get to the service and back to us" but I'm fine with widening it slightly to be cautious.